hermeto/interface/cli.py

```
@@ -174,6 +174,11 @@ def main(  # noqa: D103; docstring becomes part of --help message
         case_sensitive=False,
         help="Set log level.",
     ),
+    mode: Mode = typer.Option(
+        Mode.STRICT,
+        "--mode",
+        help="Set input expectations and requirements validation mode.",
```

I'm trying to think how to make this a little more clear without making it too long :sweat_smile:

What I feel like we want to convey is: "Set best‑practice validation mode: strict fails fast, while permissive converts violations to warnings, continues, and potentially reduces the accuracy of the SBOM".

---

How about:
`Input validation mode, 'strict' treats all violations as fatal errors, 'permissive' just audits them`

---

I updated the README.md with a detailed explanation.

> "Set best‑practice validation mode: strict fails fast, while permissive converts violations to warnings, continues, and potentially reduces the accuracy of the SBOM"
> Input validation mode, 'strict' treats all violations as fatal errors, 'permissive' just audits them

If we don't want to add more modes, then I guess it's fine. Otherwise, the help message will get very long.

---

The current wording is still pretty vague to me, okay, let's try another attempt:

_treat input requirement violations as errors|warnings (the latter may affect SBOM accuracy)_

^Still too long? :)